hs/src/Reach/Simulator/Core.hs

```diff
@@ -504,10 +504,9 @@ instance Interp DLExpr where
       let new_nw_ledger = M.insert 0 (M.delete ev m) map_ledger
       setGlobal $ e {e_ledger = new_nw_ledger }
       return V_Null
-    -- TODO: new stuff
     DLE_TimeOrder _at _assoc_maybe_arg_vars -> return V_Null
-    DLE_GetContract _at -> impossible "undefined"
-    DLE_GetAddress _at -> impossible "undefined"
+    DLE_GetContract _at -> V_Address <$> l_acct <$> getMyLocalInfo
```

Comment:
Contract are not necessarily addresses and it might be a good idea to introduce a new value, `V_Contract`, for them in the future to ensure that they can't be mixed at run time

Reply:
noted